spec/mailers/renewal_mailer_spec.rb

```diff
@@ -26,27 +26,27 @@
     end
 
     it "includes the correct template in the body" do
-      expect(mail.body.encoded).to include("Until our online renewal service is launched, you’ll need to register your exemptions again.")
+      expect(mail.body.parts[0].body.encoded).to include("Until our online renewal service is launched, you’ll need to register your exemptions again.")
```

Comment:
Adding an attachment cause the email to become a multipart email, and in order to get the correct part body, this addition was necessary.